src/libs/actions/IOU.js

```diff
@@ -294,6 +308,7 @@ function buildOnyxDataForMoneyRequest(
     isNewChatReport,
     isNewIOUReport,
 ) {
+    const isDistance = TransactionUtils.isDistanceRequest(transaction);
```

Comment:
Can we use `TransactionUtils.isScanRequest` instead?

Reply:
Sure